Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py

```diff
@@ -3287,6 +3287,15 @@ def get_email_address(eml, entry):
     if addresses:
         res = [item[1] for item in addresses]
         res = ', '.join(res)
+        if not re.search(REGEX_EMAIL, res) or len(addresses) > 1 and entry == 'from':
+            # this condition refers only to ['from'] header that does not have a valid email or have more then 1
+            # fixed an issue where email['From'] had '\r\n'.
+            # in order to solve, used replace_header() on email object,
+            # and did again get_all() on the new format of ['from']
+            original_value = eml['from']
+            eml.replace_header('from', ' '.join(eml["from"].splitlines()))
+            res = get_email_address(eml, entry)
+            eml.replace_header('from', original_value)  # replace again to the original header (keep on BC)
```

Comment:
Can you explain what is the difference from before? cause I see we did have this code before later in the flow.

Reply:
Before that, we only checked if no valid email address was found
Now we're also checking to see if there's more than one email address (which's wrong)